packages/cozy-harvest-lib/test/components/__snapshots__/TriggerManager.spec.js.snap

```diff
@@ -123,6 +123,7 @@ exports[`TriggerManager when given an oauth konnector should redirect to OAuthFo
       "handleAccountUpdated": [Function],
       "handleJobUpdated": [Function],
       "handleTwoFA": [Function],
+      "konnector": undefined,
```

Comment:
Is it normal ?

Reply:
I see no tests testing the fix, could you add one please ?

Reply:
It was not normal, I fixed the unit tests and added one unit test to check that flow knows the konnector.